packages/spark-core/base/inputs/_text-input.scss

```diff
@@ -0,0 +1,97 @@
+// ==================================================================
+// Input Component
+// ==================================================================
+.sprk-b-TextInput {
+  font-size: $font-size-body-two;
+  font-weight: $font-weight-body-two;
+  line-height: $line-height-body-two;
+  outline: 0;
+  padding: 3px $space-s;
+  box-shadow: $input-box-shadow;
+}
+
+.sprk-b-TextInput[disabled] {
+  background-color: $gray-light;
+  box-shadow: $input-disabled-box-shadow;
+  color: $input-gray-medium;
+}
+
+.sprk-b-TextInput--date-picker,
+.sprk-b-TextInput--monetary {
+  position: relative;
+}
+
+.sprk-b-DatePicker__icon,
+.sprk-b-TextInput--monetary::before {
+  position: absolute;
+  z-index: $layer-height-xs;
+}
+
+.sprk-b-DatePicker__icon {
+  margin-top: $input-icon-offset-y;
+  margin-left: $input-icon-offset-x;
+}
+
+.sprk-b-TextInput--monetary::before {
+  content: '$';
+  display: block;
+  font-size: 1rem;
+  margin-top: $input-text-icon-offset-y;
+  margin-left: $input-text-icon-offset-x;
+  line-height: 1;
+}
+
+.sprk-b-TextInput[readonly] {
+  color: $white;
+}
+
+.sprk-b-HelperText,
+.sprk-b-ErrorText {
+  font-size: $font-size-body-four;
+  font-weight: $font-weight-body-four;
+  line-height: $line-height-body-four;
+}
+
+.sprk-b-HelperText {
+  color: $input-gray-medium;
+}
+
+.sprk-b-ErrorText {
+  font-weight: bold;
+}
+
+.sprk-b-TextArea {
+  min-height: 125px;
+  margin: $space-stack-m;
+}
+
+.sprk-b-Fieldset {
+  padding: 0;
```

Comment:
Don't need this because Scales exposes the $fieldset-padding variable that we can set to 0 in our settings file.